src/Objs/SearchResults/SearchResultItem.js

```diff
@@ -12,8 +12,9 @@ const PreviewImage = Scrivito.connect(({ item }) => {
     item.get("navigationBackgroundImage") ||
     item.get("titleImage") ||
     item.get("image");
-  if (image instanceof Scrivito.Binary) return null;  
-  if (!image || isVideoObj(image)) return null;
+  if (!image) return null;
+  if (!(image instanceof Scrivito.Obj)) return null;
```

Comment:
FYI: To handle the "falsy" case separately is not actually required. Worst case (works also for `null`, `false`, `""`, and `0` (and likely for any other falsy JS thingy which is never passed here actually)):
```
undefined instanceof Object 
// => false
```